ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -152,6 +175,22 @@ protected void onScrollChanged(int x, int y, int oldX, int oldY) {
   }
 
   @Override
+  public View hitTest(float[] coordinates) {
```

Comment:
Yea, good idea, I just checked and AOSP does use getChildDrawingOrder when dispatching a touch event to subviews.